--- kpp_calc.F.orig	2026-08-19 00:00:00.000000000 -0800
+++ kpp_calc.F	2026-08-19 00:00:00.000000000 -0800
@@ -596,6 +596,13 @@
        ENDDO
       ENDDO

+c-------------------------------------------------------------------
+c     Output KPP inputs and outputs for Python validation
+c-------------------------------------------------------------------
+      CALL KPP_OUTPUT_VALIDATION(
+     I     myIter, bi, bj, myThid )
+
 #ifdef KPP_SMOOTH_VISC
 c     horizontal smoothing of vertical viscosity
       DO k = 1, Nr
